templates/web/pages/bmviii_home_temp.tmpl

```diff
@@ -0,0 +1,265 @@
+{{ define "bmviii home temp" }}
+{{ template "head" . }}
+
+{{ template "bmviii navbar" . }}
+
+{{ template "flashes" .Flashes }}
+{{ template "mlh badge" }}
+
+<section class="section has-background-image" style="background-color: #A7C3FF; position: relative; padding-top: 0px; padding-bottom: 0px; padding-left: 0px; padding-right: 0px">
+    <img alt="BM buildings" src={{ static_path "bmviii_lp_buildings.svg" }} style="vertical-align: bottom">
+    <div class="container bmviii-front-logo logo-div" style="top: 7%;">
+        <img class="logo-img" alt="Boilermake" src={{ static_path "bmviii_boilermake_logo.svg" }}>
+    </div>
+    <div class="container bmviii-front-logo" style="top: 23%; padding-left: 2.5%">
+        <img alt="Lay back and hack" src={{ static_path "bmviii_lay_back_and_hack.svg" }} width="80%">
+    </div>
+    <div class="container bmviii-front-logo" style="top: 37%; padding-left: 5%">
+        <a href="https://forms.gle/ugmBaA4bHvm6rwjF7">
+            <img alt="Apply Now!" src={{ static_path "bmviii_apply_now.svg" }} width="65%">
+        </a>
+    </div>
+    <div class="container bmviii-front-logo" style="top: 60%; padding-left: 2.3%">
+        <img alt="January 22-24, 2021" src={{ static_path "bmviii_date.svg" }} width="85%">
+    </div>
+    <div class="container bmviii-front-logo" style="top: 70%; padding-left: 5%">
+        <img alt="Purdue University" src={{ static_path "bmviii_purdue_text.svg" }} width="75%">
+    </div>
+</section>
+
+<section class="section has-background-image" id="why" style='background-color: #FF8E9E; background-image: url({{ static_path "bmviii_why_bm_background.svg" }}); padding-bottom: 125px; padding-left: 0; padding-right:0;'>
+	<div class="container" style="padding: 0 1.5rem;">
+		<div class="columns is-centered content">
+			<div class="column is-2 bmviii-section-header" style="margin-left: 48px; margin-right: 12px; margin-top:72px">
+				<img alt="Why BoilerMake?" src={{ static_path "bmviii_why_bm_text.svg" }}>
+			</div>
+		</div>
+	</div>
+	<div class="container" style="display: flex; justify-content: flex-end; padding-top: 48px; margin-right: 0px;">
+	    <div class="column is-full">
+            <div class="box content" style="width:1218px;">
```

Comment:
What's up with this width rule?

Reply:
Bad code. I removed it when looking at it again. I'll update it in the next pr